app/helpers/socure_webhook_helper.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module SocureWebhookHelper
+end
```

Comment:
is this generated by rails? looks empty + unused, I'd vote to remove it

Reply:
Removed, along with the corresponding empty spec.